src/Symfony/Component/Cache/Adapter/AdapterInterface.php

```diff
@@ -21,4 +21,17 @@
  */
 interface AdapterInterface extends CacheItemPoolInterface
 {
+    /**
+    * {@inheritdoc}
+    *
+    * @return \Symfony\Component\Cache\CacheItem
+    */
+    public function getItem($key);
+
+   /**
+    * {@inheritdoc}
+    *
+    * return \Symfony\Component\Cache\CacheItem[]
```

Comment:
could be `@return CacheItem[]`

I've also been said once that `Foo[]` means a "numerically index set of Foo". This wouldn't be accurate here, since the indexes are the keys, and the base type is array|Traversable. Could someone confirm?